moflask/requests/auth.py

```diff
@@ -9,12 +9,14 @@
 class AuthAppClient(rest.Client):
     """A REST client for retrieving JWTs from the auth-app."""
 
+    AUTH_API_VERSION = "v1"
+
     @classmethod
     def from_app(cls, app=None):
         """Create a new instance by reading the config from the Flask app config."""
         app = app or flask.current_app
         return cls(
-            app.config["IMPACT_STACK_AUTH_APP_URL"],
+            app.config["IMPACT_STACK_AUTH_APP_URL"] + "/" + cls.AUTH_API_VERSION,
```

Comment:
Wasn’t the idea to switch to a single `IMPACT_STACK_API_URL` config anyways? This looks like a good opportunity :wink:

Reply:
Good idea. I’ve updated the PR accordingly.